tardis/model/base.py

```diff
@@ -84,6 +84,27 @@ def r_outer(self):
         return self.geometry.r_outer.values * self.geometry_units["r_outer"]
 
 
+class Radial1DGeometry:
+    """
+    Holds information about model geometry for radial 1D models.
+
+    Parameters
+    ----------
+    radius : astropy.units.quantity.Quantity
```

Comment:
Make sure the description of the doc-string matches the parameters of the __init__